thebuttonclan/pangolin
# docs: 476 documentation updates

Fixes #476 

## Proposed Changes

- Add _Getting Started_ section to **Button Theme**
- Refactor _Getting Started_ section in both themes to use common stories
- Update some base documentation with new common components

## To test

If testing locally, you'll need to run both `yarn storybook:button` and `yarn storybook:bcgov` to see changed documents.

## Learnings/notes

Due to how `.mdx` interacts with _Markdown_ and _JSX_ with regards to variables, parts of the documentation was rewritten from MD into JSX (see `stories/common/Using_the_Kit.mdx` for more). Flagged this for refactoring at a later date if/when Storybook adopts MDXv2 which resolves these in a different manor.
